paper-header-panel.css

```diff
@@ -0,0 +1,106 @@
+/**
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+*/
+
+:host {
+  @apply(--layout);
+  @apply(--layout-vertical);
+
+  display: block;
+  position: relative;
+  height: 100%;
+}
+
+#mainContainer {
+  @apply(--layout-flex);
+
+  position: relative;
+  overflow-y: auto;
+  overflow-x: hidden;
+  -webkit-overflow-scrolling: touch;
+  flex-basis: 0.0001px;
```

Comment:
don't need `flex-basis` here since `iron-flex-layout` is already setting this:
https://github.com/PolymerElements/iron-flex-layout/blob/master/iron-flex-layout.html#L87